src/chainparams.cpp

```diff
@@ -478,8 +478,10 @@ class CRegTestParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DYNA_FED].nStartTime = 1199145601; // January 1, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_DYNA_FED].nTimeout = 1230767999; // December 31, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = Consensus::BIP9Deployment::ALWAYS_ACTIVE;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = gArgs.GetArg("-con_taproot_signal_start", Consensus::BIP9Deployment::ALWAYS_ACTIVE);
```

Comment:
In 5291c0d9d9c09b3613d2fe3fb73311edab7491f3 "chainparams: add undocumented regtest/testnet only -con_taproot_signal_start option"

This is entirely unnecessary as there already exists a way to set regtest activation params via `-vbparams`. See the function `CRegTestParams::UpdateActivationParametersFromArgs`.

Reply:
Thank you! I didn't know about this.

I'm going to leave it in because this code is inherently temporary and we're already using it for internal testing, but in future we can avoid these kind of hacky undocumented options.

Reply:
Eh, also didn't know that. We did the same thing for dynafed :)